docs/ONBOARDING.md

```diff
@@ -56,7 +56,30 @@ This app uses a monorepo structure with the client application in the `client` d
 
 ## Client
 
-TODO
+### Flutter Setup
```

Comment:
I'd rather see just a reference to: https://flutter.dev/docs/get-started/install

No use duplicating content that might go out of date.